src/pages/Settings/EmailSettings.vue

```diff
@@ -627,8 +679,6 @@ export default {
 								}
 							}
 						`,
-						awaitRefetchQueries: true,
```

Comment:
Why were these two lines removed?

Reply:
Because we have the mutation below doing the same.

Reply:
Right, but these are different mutations. Don't we also need to refetch after unsubscribing here?

Reply:
Refetching is to get the updated data on the page, `pageQuery` and show it as mutations are done.
Unsubscribing implies repayments too, so for the case `Only legally required emails` we need the second mutation if we have a different value than default for repayments, so we need to refetch the page data once. makes sense?

Reply:
Oh I clicked to expand the methods and see that both mutations are in the same method, makes sense, I missed that before.